src/components/reservation/ReservationModalBody.tsx

```diff
@@ -36,23 +36,29 @@ import {
   getAuthorLine
 } from "./helper";
 import UseReservableManifestations from "../../core/utils/UseReservableManifestations";
+import { GroupListItem } from "../material/MaterialPeriodicalSelect";
 
 export const reservationModalId = (faustId: FaustId) =>
   `reservation-modal-${faustId}`;
 
 type ReservationModalProps = {
   mainManifestation: Manifestation;
   parallelManifestations?: Manifestation[];
+  periodicalSelect?: GroupListItem | null;
 };
 
 const ReservationModalBody = ({
   mainManifestation,
   mainManifestation: { pid, materialTypes, titles, edition },
-  parallelManifestations
+  parallelManifestations,
+  periodicalSelect
 }: ReservationModalProps) => {
   const mainManifestationType = getManifestationType(mainManifestation);
   const { reservableManifestations } = UseReservableManifestations({
-    manifestations: parallelManifestations ?? [mainManifestation],
+    manifestations:
+      parallelManifestations && parallelManifestations.length > 0
```

Comment:
The type `ReservationModalProps` says that `ReservationModalProps.parallelManifestations` either is `undefined` or `Manifestation[]`. It should not be necessary to check for: `parallelManifestations.length > 0`

Reply:
I think the empty array is the same as true and therefore it will not select mainManifestation